### ISSUE TITLE
Compilation errors relating to the protobuf include files.

### ISSUE THREAD

**Issue by negiyas** [onnx/onnx-mlir#1044]
I got the following compile errors to build the latest main branch of onnx-mlir  (commit 90d33c03d01356668318b386098667ddcd04d644 ).
It seems that we need to include google/protobuf/port_def.inc file in the protobuf package.

```
...
[ 40%] Building CXX object src/Dialect/ONNX/CMakeFiles/OMONNXOps.dir/ShapeInfer\
ence/Squeeze.cpp.o
In file included from /home/negishi/src/dlc.git/onnx-mlir/third_party/onnx/onnx\
/onnx_pb.h:52:0,
                 from /home/negishi/src/dlc.git/onnx-mlir/src/Runtime/PyExecuti\
onSession.cpp:16:
/home/negishi/src/dlc.git/onnx-mlir/build/third_party/onnx/onnx/onnx-ml.pb.h:10\
:10: fatal error: google/protobuf/port_def.inc: No such file or directory
 #include <google/protobuf/port_def.inc>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
src/Runtime/CMakeFiles/PyRuntime.dir/build.make:82: recipe for target 'src/Runt\
ime/CMakeFiles/PyRuntime.dir/PyExecutionSession.cpp.o' failed
```
protoc version 3.11.4 is in the PATH, and according to output of cmake attached below, it seems that cmake can find the protobuf include path correctly.
Should we set some environment variables or arguments for cmake command to avoid the errors?

```
Output of cmake
...
-- ******** Summary ********
--   CMake version             : 3.17.5
--   CMake command             : /home/negishi/use/cmake-3.17.5/bin/cmake
...
--   ONNXIFI_ENABLE_EXT        : OFF
--
--   Protobuf compiler         : /home/negishi/use/protobuf/bin/protoc
--   Protobuf includes         : /home/negishi/use/protobuf-3.11.4/include
--   Protobuf libraries        : /home/negishi/use/protobuf-3.11.4/lib/libprotobuf.so
...

```

**Comment by MaeThird** [comment]
I solved this by adding protobuf's directory(in my case it's  /usr/local/include)directly in CMakeLists.txt like
 ```set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -I/usr/local/include")```

**Comment by gongsu832** [comment]
@negiyas Did you by any chance updated either python or protobuf? If yes you need to rebuild `third_party/onnx`.
```
cd third_party/onnx
rm -rf .eggs .setuptools-cmake-build build dist onnx.egg-info
python3 setup.py install
```
That's what I do typically.

**Comment by negiyas** [comment]
@MaeThird I can avoid the errors by adding protobuf's directory in CMakeLists.txt. Thank you for the comments. 
@gongsu832 I executed "python setup.py install" at third_party_onnx, but it cannot fix the issue.
In my case, I could built onnx-mlir before, and I did not update python or protobuf after that.
I clone the onnx-mlir project from scratch by "git clone --recursive https://github.com/onnx/onnx-mlir.git".
If you know some checking points to investigate this issue, I will try it. Thank you!
